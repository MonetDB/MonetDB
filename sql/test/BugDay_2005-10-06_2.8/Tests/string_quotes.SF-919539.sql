select 'a' 'a';
select 'a''a';
