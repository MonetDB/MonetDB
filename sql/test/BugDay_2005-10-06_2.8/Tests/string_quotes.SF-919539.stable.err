stderr of test 'string_quotes.SF-919539` in directory 'sql/test/BugDay_2005-10-06_2.8` itself:


# 15:10:40 >  
# 15:10:40 >  Mtimeout -timeout 180 Mserver "--config=/home/sjoerd/src/stable/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/home/sjoerd/src/stable/lib/MonetDB:/home/sjoerd/src/stable/lib/bin" --set "gdk_dbfarm=/home/sjoerd/src/stable/var/MonetDB/dbfarm" --set "sql_logdir=/home/sjoerd/src/stable/var/MonetDB/log" --set mapi_port=38305 --set sql_port=40913 --set xquery_port=59727 --set monet_prompt= --trace "--dbname=mTests_src_test_BugDay_2005-10-06_2.8" --dbinit="module(sql_server); sql_server_start();" ; echo ; echo Over..
# 15:10:40 >  


# 15:10:41 >  
# 15:10:41 >  Mtimeout -timeout 60 MapiClient -lsql -u monetdb -P monetdb --host=localhost --port=40913  < string_quotes.SF-919539.sql
# 15:10:41 >  


# 15:10:41 >  
# 15:10:41 >  Done.
# 15:10:41 >  

