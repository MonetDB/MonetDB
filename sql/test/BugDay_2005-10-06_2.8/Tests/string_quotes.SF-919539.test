query T rowsort
select 'a' 'a'
----
aa

query T rowsort
select 'a''a'
----
a'a

