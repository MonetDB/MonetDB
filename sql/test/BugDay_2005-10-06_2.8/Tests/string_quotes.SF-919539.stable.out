stdout of test 'string_quotes.SF-919539` in directory 'sql/test/BugDay_2005-10-06_2.8` itself:


# 12:37:30 >  
# 12:37:30 >  Mtimeout -timeout 180 Mserver "--config=/ufs/fabian/scratch/monetdb/current/program-x86_64/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/ufs/fabian/scratch/monetdb/current/program-x86_64/lib/MonetDB:/ufs/fabian/scratch/monetdb/current/program-x86_64/lib/bin" --set "gdk_dbfarm=/ufs/fabian/scratch/monetdb/current/program-x86_64/var/MonetDB/dbfarm" --set "sql_logdir=/ufs/fabian/scratch/monetdb/current/program-x86_64/var/MonetDB/log" --set mapi_port=38933 --set sql_port=45579 --set xquery_port=52530 --set monet_prompt= --trace "--dbname=mTests_src_test_BugDay_2005-10-06_2.8" --dbinit="module(sql_server); sql_server_start();" ; echo ; echo Over..
# 12:37:30 >  

# Monet Database Server V4.9.3
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for x86_64-redhat-linux-gnu/64bit with 64bit OIDs; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.




#select 'a' 'a';
% .%2 # table_name
% %2 # name
% char # type
% 2 # length
[ "aa"	]
#select 'a''a';
% .%2 # table_name
% %2 # name
% char # type
% 3 # length
[ "a'a"	]

# 12:37:31 >  
# 12:37:31 >  Done.
# 12:37:31 >  

